src/projects/aws-eks/index.ts

```diff
@@ -62,36 +102,45 @@ const cluster = new eks.Cluster(`ai-eks-cluster-${pulumiStack}`, {
     stack: pulumiStack
   },
   instanceRoles: [role],
+  roleMappings: [
+    {
+      roleArn: identity.adminsIamRoleArn,
+      groups: ["system:masters"],
+      username: "pulumi:admins",
+    },
+    {
+      roleArn: identity.devsIamRoleArn,
+      groups: ["pulumi:devs"],
+      username: "pulumi:alice",
+    },
+  ],
   providerCredentialOpts: {
-    profileName: config.get('aws:profile')
+    profileName: awsConfig.get('profile')
   }
 });
 
-// Create a simple AWS managed node group using a cluster as input.
-const managedNodeGroup = eks.createManagedNodeGroup(
-  `ai-eks-mng--${pulumiStack}`,
-  {
-    cluster: cluster,
-    nodeGroupName: `ai-eks-mng--${pulumiStack}`,
-    nodeRoleArn: role.arn,
-    labels: { ondemand: 'true' },
-    tags: { 
-      org: 'pulumi', 
-      managedBy: 'aitomatic',
-      stack: 'pulumiStack'
-    },
-    scalingConfig: {
-      minSize: 2,
-      maxSize: 20,
-      desiredSize: 2
-    },
-  },
-  cluster
-);
-
 // Export the cluster's kubeconfig.
 export const kubeconfig = cluster.kubeconfig;
 
+//Export profile to add to ~/.aws/config
+export const awsProfileSnippet = pulumi.interpolate`
+[profile ${pulumiStack}]
+region = ${awsRegion}
+output = json
+role_arn = ${identity.adminsIamRoleArn}
+source_profile = default
```

Comment:
this one should be `awsConfig.get('profile')` as the default profile might not correctly using Aitomatic